Assets/Scripts/Player/Damage.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+//This code is used for testing the Health of limbs and such resulting from specific types of damage
+//In order to change the targetted location use 'Q' and 'E'
+//In order to change the damage tpye use 'A' and 'D'
+//In order to change the amount of damage being inflicted use 'O' and 'P'
+//In order to deal damage and get an output, press 'W'.
+//The target of this script sould be the player being attacked
+
+public class Damage : MonoBehaviour
+{
+    public GameObject playerTarget;
+    Health attack;
+
+    //used for testing
+    public static int currentTarget = 0;
+    public static int currentDmgType = 0;
+    public static float dmg = 0.0f;
+
+    public enum DamageType
+    {
+        brute,
+        burn,
+        toxic,
+        suffocation
+    }
+
+    enum SectionTarget
+    {
+        torso,
+        head,
+        leftArm,
+        rightArm,
+        leftLeg,
+        rightLeg
+    }
+
+    // Start is called before the first frame update
+    void Start()
+    {
+    }
+
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Q) && currentTarget > 0)

```

Comment:
These code blocks should be condensed into a single if() for each keycode followed by a nested if() for the second parameters, to reduce the amount of times GetKeyDown() is called.